nucypher/cli/painting.py

```diff
@@ -173,36 +182,149 @@ def paint_known_nodes(emitter, ursula) -> None:
-Published Stakes ......... {staking_agent.get_staker_population()}
-Gas Price ................ {Web3.fromWei(blockchain.client.gas_price, 'gwei')} Gwei
+Stakers population ....... {staking_agent.get_staker_population()}
+   Confirmed ............. {len(confirmed)}
+   Pending confirmation .. {len(pending)}
+   Inactive .............. {len(inactive)}
+
     """
+
+    sep = '-' * 45
+    emitter.echo(sep)
+    emitter.echo(contracts)
+    emitter.echo(sep)
+    emitter.echo(blockchain)
+    emitter.echo(sep)
+    emitter.echo(staking)
+    emitter.echo(sep)
+
+
+def paint_deployer_contract_status(emitter, administrator) -> None:
+
+    blockchain = BlockchainInterfaceFactory.get_interface()
+    token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=administrator.registry)
+
+    sep = '-' * 45
+    emitter.echo(sep)
+
+    contract_payload = f"""
+
+* Web3 Provider
+====================================================================
+
+Provider URI ............. {blockchain.provider_uri}
+Registry  ................ {administrator.registry.filepath}
+
+* Standard Deployments
+=====================================================================
+
+NucypherToken ........... {token_agent.contract_address}
+    ~ Ethers ............ {blockchain.client.get_balance(token_agent.contract_address)} wei
+    ~ Tokens ............ {token_agent.get_balance(token_agent.contract_address)} wei"""
     emitter.echo(contract_payload)
-    emitter.echo(network_payload)
+
+    banner = """
+* Proxy-Contract Deployments
+====================================================================="""
+    emitter.echo(banner)
+
+    for contract_deployer_class in administrator.dispatched_upgradeable_deployer_classes:
+        try:
+            bare_contract = blockchain.get_contract_by_name(name=contract_deployer_class.contract_name,
+                                                            proxy_name=DispatcherDeployer.contract_name,
+                                                            registry=administrator.registry,
+                                                            use_proxy_address=False)
+
+            dispatcher_deployer = DispatcherDeployer(registry=administrator.registry,
+                                                     target_contract=bare_contract,
+                                                     deployer_address=administrator.deployer_address,
+                                                     bare=True)  # acquire agency for the dispatcher itself.
+
+            agent = contract_deployer_class.agency(registry=administrator.registry, contract=bare_contract)
+
+            proxy_payload = f"""
+{agent.contract_name} .... {bare_contract.address}
+    ~ Owner .............. {bare_contract.functions.owner().call()}
+    ~ Ethers ............. {blockchain.client.get_balance(dispatcher_deployer.contract_address)} wei
+    ~ Tokens ............. {token_agent.get_balance(dispatcher_deployer.contract_address)} wei
```

Comment:
Same comment here regarding `nunits` instead of `wei`. Other spot on line 289 below as well